src/Cli/Microsoft.DotNet.FileBasedPrograms/Microsoft.DotNet.FileBasedPrograms.Package.csproj

```diff
@@ -57,4 +57,24 @@
   </ItemGroup>
 
   <Import Project="..\Microsoft.DotNet.FileBasedPrograms\Microsoft.DotNet.FileBasedPrograms.projitems" Label="Shared" />
+
+  <!-- Include resx and xlf files in the package (once, not per-TFM) -->
+  <ItemGroup>
+    <None Include="$(MSBuildThisFileDirectory)FileBasedProgramsResources.resx">
+      <Pack>true</Pack>
+      <PackagePath>contentFiles\cs\any\FileBasedProgramsResources.resx</PackagePath>
+    </None>
+    <None Include="$(MSBuildThisFileDirectory)xlf\*.xlf">
+      <Pack>true</Pack>
+      <PackagePath>contentFiles\cs\any\xlf</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <!-- Exclude the auto-generated .cs file from the resx from the package -->
+  <Target Name="ExcludeGeneratedResxFromPackage" BeforeTargets="GenerateNuspec">
+    <ItemGroup>
+      <_PackageFiles Remove="@(_PackageFiles)" Condition="$([System.String]::Copy('%(_PackageFiles.Identity)').EndsWith('FileBasedProgramsResources.cs'))" />
```

Comment:
Where exactly is this generated file located originally? Somewhere in `obj`/`bin` folder?

Anyway, perhaps a simpler way to exclude it would be something like

```csproj
<Compile Update="/path/to/the/file.cs" Pack="false" />
```

(where the path could be perhaps even something like `**/FileBasedProgramsResources.cs`)

(still might need to live in a Target though if the original item we are updating was defined in a Target as well)